working-examples/css-44px-target-size/index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<title>Working example - Using CSS to ensure sufficient target size</title>	
+<style>
+	:root {
+		font-size: 100%;
+		box-sizing: border-box;
+	}
+
+	*,*::after,*::before {
+		box-sizing: inherit;
+	}
+
+	body {
+		background:#fff;
+		color:#000;
+		font: 1rem/1.5 system-ui;
+	}
+
+	.visually-hidden {
+		clip: rect(0, 0, 0, 0);
+    clip-path: inset(50%);
+    height: 1px;
+    overflow: hidden;
+    position: absolute;
+    white-space: nowrap;
+    width: 1px;
+  }
+
+	.pagination-list{
+		display: flex;
+		flex-wrap: wrap;
+		gap:0.5em;
+		list-style: none;
+		padding:0;        
+	}
+
+	.pagination-list li{
+		background:#B9D6E5;
+		border:1px solid #183544;
+		padding:0.25em;
```

Comment:
```suggestion
```

Super editorial, but I find that the `li`’s padding in [C42](https://deploy-preview-4685--wcag2.netlify.app/working-examples/css-24px-target-size/) helps tell the story of why it’s acceptable in [2.5.8](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html) for a link’s interactive footprint to be smaller than 24×24 px as long as it has sufficiently additive “dead zone” around it.

In this new technique for [2.5.5](https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced.html), the visible 0.25em dead zone feels like a red herring.